nixos/modules/system/boot/luksroot.nix

```diff
@@ -414,12 +509,6 @@ in
                     description = "The filesystem of the unencrypted device.";
                   };
 
-                  mountPoint = mkOption {
-                    default = "/crypt-storage";
-                    type = types.str;
-                    description = "Path where the unencrypted device will be mounted during early boot.";
-                  };
-
                   path = mkOption {
```

Comment:
Note: (*nothing to change here*)

The `path` is written as if an absolute path, bit this seems to be relative to the root *of the yubikey*. The semantics of the option *did not change with the updated code*. The only difference I can see is that the mount point is now hard-coded to `/crypt-storage` instead of being an option, but *the code will look at `/crypt-storage/crypt-storage/default` using the default options, both previously and now.

I'm leaving this note as I was about to ask for changes in the confusion this caused.